options-manager.js

```diff
@@ -191,10 +192,20 @@ function groupConfigs(paths, baseOptions, overrides) {
 	return arr;
 }
 
+function getIgnores(opts) {
+	const gitignore = parseGitignore('.gitignore');
+
+	opts.ignores = DEFAULT_IGNORE.concat(opts.ignores || []);
+	opts.ignores = opts.ignores.concat(gitignore || []);
```

Comment:
No need for multiple concat statement. `Array#concat` support multiple arguments.